gcp/appengine/frontend_handlers.py

```diff
@@ -345,11 +345,16 @@ def osv_get_ecosystems():
                 key=str.lower)
 
 
-# TODO: Figure out how to skip cache when testing
-@cache.instance.cached(
-    timeout=24 * 60 * 60, key_prefix='osv_get_ecosystem_counts')
+@cache.smart_cache("osv_get_ecosystem_counts", timeout=24 * 60 * 60)
```

Comment:
Could we use https://pypi.org/project/humanize/ to make this timeout a little more human-friendly?